src/cli/cli.cpp

```diff
@@ -133,6 +132,11 @@ const struct Command Interpreter::sCommands[] =
     { "thread", &Interpreter::ProcessThread },
     { "version", &Interpreter::ProcessVersion },
     { "whitelist", &Interpreter::ProcessWhitelist },
+
+#ifdef OPENTHREAD_FTD_CLI
+    { "delaytimermin", &Interpreter::ProcessDelayTimerMin},
+    { "joinerport", &Interpreter::ProcessJoinerPort },
+#endif
```

Comment:
I believe we want these to remain in alphabetical order, so can you just put the `#ifdef` around the original locations in the array?